### ISSUE TITLE
Missing package: p4p

### ISSUE THREAD

**Issue by mdavidsaver** [piwheels/packages#117]
Package name: p4p
Issue type: Missing package
Link to PyPI page: https://pypi.org/project/p4p
Link to piwheels page: https://www.piwheels.org/project/p4p/
Version: 3.5.0
Python version: 2.7 and >=3.4
I am the maintainer: Yes
More information:

I see that only py3.4 wheels are being published for my P4P package.  Is there a log available to show what is failing for the 3.5 and 3.7 builds?

FYI. the dependency [epicscorelibs](https://www.piwheels.org/project/epicscorelibs/) is successfully built for cp34m, cp35m, and cp37m.

**Comment by bennuttall** [comment]
Here's the very end of the build output (which is 20k lines!!) from 1.5.0 on Buster: http://paste.debian.net/1148639/

It's complaining about a deprecated numpy version, but that doesn't seem to be a fatal error. Here it is:

```
src/gwchannel.cpp:3:10: fatal error: _gw.h: No such file or directory
2020-05-18T21:52:50,843    #include "_gw.h"
2020-05-18T21:52:50,844             ^~~~~~~
2020-05-18T21:52:50,844   compilation terminated.
```

I'm not sure how it succeeded on Jessie. What are the build dependencies? If there's an apt package that's needed, we can install it and try again.

**Comment by mdavidsaver** [comment]
The header `_gw.h` should be emitted by the cython code generator.  It would be helpful to see the top of the log to see which version (if any) is being installed.

**Comment by mdavidsaver** [comment]
In particular, if this line appears in the log.

```
Cython not installed.  Trying to use dummy cythonize()
```

**Comment by mdavidsaver** [comment]
Ok, I think I see what the problem is. My MANIFEST.in is incomplete.  I'll be pushing a new version soon.

**Comment by mdavidsaver** [comment]
Some progress.  Now that py3.5 build succeeds.

**Comment by bennuttall** [comment]
3.5.1 log on cp37m: http://paste.debian.net/1148868/

**Comment by mdavidsaver** [comment]
I think I see what is going on.  Again, my issue.  Looks like a side effect of mixing portability testing and production builds in the same set of CI jobs.

Some of my users are a bit behind the times.  In order to support old RHEL, I need to build py2.7 wheels against old numpy.  I do this with a set of `requirements-*.txt` files based on Debian releases.  Debian 8 in this case.  This is also pulling in an older Cython which doesn't support py3.7.  Combine this will a race to see which job uploads the source dist first, and the fact that the py2.7 jobs appear first in `.travis.yml`, and the uploaded source tar almost always contains code generated by an older Cython.

**Comment by mdavidsaver** [comment]
I think I have the sources straightened out, and the piwheels builds now all succeed.  Thanks for providing this service.

A couple of notes.  I had to install the `libatlas3-base` debian/raspbian package to get `libf77blas.so.3` which is needed by the numpy build from pywheels.

Also, since I found it surprisingly difficult to find an example, the recipe for PIP to download wheel files for an offline pi-zero:

```sh
mkdir dl
python3 -m pip download -d dl/ \
 --extra-index-url https://www.piwheels.org/simple \
 --python-version 37 \
 --implementation cp \
 --platform linux_armv6l \
 --abi cp37m \
 --only-binary=:all: \
 p4p
```
